spec/lib/miq_automation_engine/service_methods/miq_ae_service_spec.rb

```diff
@@ -3,9 +3,21 @@
 module MiqAeServiceSpec
   describe MiqAeMethodService::MiqAeService do
     before(:each) do
-      @domain = 'TEST_DOMAIN'
-      identifiers = {:domain => @domain, :namespace => 'EVM',
-                          :class  => 'AUTOMATE', :instance => 'test1', :method => 'test'}
+      MiqAeDatastore.reset
+      @domain = 'Bedrock'
+      Tenant.seed
+      @root_tenant = Tenant.root_tenant
```

Comment:
future reference:

`@root_tenant = Tenant.seed`
don't do now (unless you are going back in)